.travis.yml

```diff
@@ -1,11 +1,13 @@
-language: python
-
 service:
```

Comment:
If we're switching to Github Actions, I suggest we get rid of this - having to maintain two pipelines is just creating more work!